src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -622,7 +625,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
           case Some(v) => Success(v)
           case _ => Failure(new WdlExpressionException("Unknown error"))
         }
-        val coercedValue = value.flatMap(x => taskInput.wdlType.coerceRawValue(x))
+        val coercedValue = value.map(x => taskInput.wdlType.coerceRawValue(x).get)
```

Comment:
I'm almost positive these two things were bad merges as I've had to do a bunch of them over time. I can't imagine why else I'd make these changes, I just haven't confirmed